### ISSUE TITLE
Cannot find module 'lodash'

### ISSUE THREAD

**Issue by brucedjones** [airbnb/ts-migrate#126]
I get this error when running any ts-migrate command eg

```bash
$ npx ts-migrate -- rename . -s "src/**/*.js"
npx: installed 478 in 19.015s
Cannot find module 'lodash'
```

I've tried installing lodash globally, it is already installed in the repo i'm running this in.

**Comment by oshabaeva** [comment]
I have the same issue. I do have @types/lodash

**Comment by edsrzf** [comment]
It looks like this is because jscodeshift requires lodash, but does not list it as a dependency. https://github.com/facebook/jscodeshift/blob/57a9d9c731429da624655472290a7b4802e7e8d7/parser/tsx.js#L11

We don't otherwise depend on lodash, except through dev dependencies. 😕 

We could probably work around this by adding a dependency on lodash. 😢

**Comment by lencioni** [comment]
Relevant issues:

- https://github.com/facebook/jscodeshift/issues/418
- https://github.com/facebook/jscodeshift/issues/426
- https://github.com/facebook/jscodeshift/pull/427
- https://github.com/facebook/jscodeshift/pull/432

**Comment by jplaisted** [comment]
I get the same error, even after a `yarn global add lodash` / `npm install -g lodash`. Seems like this might be resolved with a bump of `jscodeshift`?

**Comment by Rudeg** [comment]
This should be resolved in `0.1.21`
